release/sbin/clear-all.sh

```diff
@@ -35,6 +35,19 @@ rm_dir() {
 if [[ ${OPENMLDB_MODE} == "standalone" ]]; then
   rm -rf standalone_db standalone_logs
 else
+  conf_file="conf/tablet.flags.template"
+  dirname=""
+  while IFS= read -r line
+  do
+    if echo "$line" | grep -q '^#'; then
+      continue
+    fi
+    if echo "$line" | grep -v "zk_root_path" | grep -q "root_path" ||
+        echo "$line" |  grep -q "openmldb_log_dir"; then
+      cur_dirname=$(echo "${line}" | awk -F '=' '{print $2}')
```

Comment:
I'm afraid the scripts are really unsafe, think the result of 
```sh
echo 'root_path=/ abc' | awk -F '=' '{print $2}'
```

is `/` and `abc`, it will erase the whole system

Reply:
If possible, it should never runs under root